wagtail/admin/templates/wagtailadmin/shared/side_panels/preview.html

```diff
@@ -54,7 +54,9 @@ <h3 class="preview-panel__error-title">
         </div>
 
         <div class="preview-panel__wrapper">
-            <iframe loading="lazy" title="{% trans 'Preview' %}" class="preview-panel__iframe" data-preview-iframe aria-describedby="preview-panel-error-banner"></iframe>
+            <iframe loading="lazy" title="{% trans 'Preview' %}" class="preview-panel__iframe" data-preview-iframe aria-describedby="preview-panel-error-banner">
+                <div{# Add placeholder element to support styling content when iframe has loaded #}/>
```

Comment:
Please don’t use self-closing div: https://jakearchibald.com/2023/against-self-closing-tags-in-html/

and can we please just put the comment in its own line? Thank you

Reply:
@laymonage You'll be happy to know that curlylint raises a warning with self-closing divs

Reply:
Thanks Sage. Sorry for the rushed suggestion on my part.